app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardsManagementFragment.kt

```diff
@@ -72,9 +79,33 @@ class CreditCardsManagementFragment : SecureFragment() {
 
     override fun onResume() {
         super.onResume()
+        activity?.window?.setFlags(
+            WindowManager.LayoutParams.FLAG_SECURE,
+            WindowManager.LayoutParams.FLAG_SECURE
+        )
         showToolbar(getString(R.string.credit_cards_saved_cards))
     }
 
+    /**
+     * If we pause this fragment, we want to pop users back to the settings page to reauthenticate.
+     */
+    override fun onPause() {
```

Comment:
Maybe this should also go into `SecureFragment`

Reply:
Please provide a response before resolving a conversation if no action was taken. I wouldn't be leaving a comment if there was no merit to it. Best case scenario would be to think about it and decide if changed were warranted or provide some thoughtful feedback to comment. If it's decided we wanted to do this as a followup, leaving the comment unresolve and linking to it back in the followup ticket would also been a better option.

Reply:
Redirecting could go in the SecureFragment, but since there are also specific things related to logins and credit cards I think this should stay in the respective fragments. Re-writing the secure fragment is a different piece of work than spinning up auth for credit cards.